openquake/sep/classes.py

```diff
@@ -319,7 +319,7 @@ class RashidianBaise2020Liquefaction(SecondaryPeril):
     to binary output via the predefined probability threshold.
     """
 
-    outputs = ["LiqProb", "LiqOccur", "LSE"]
+    outputs = ["LiqProb", "LiqOccur", "LqSE"]
```

Comment:
If we are using Lq for liquefaction, can we also change LiqProb and LiqOccur to LqProb and LqOccur to be consistent?